src/System.Management.Automation/resources/ExtendedTypeSystem.resx

```diff
@@ -156,6 +156,12 @@
   <data name="MethodArgumentCountException" xml:space="preserve">
     <value>Cannot find an overload for "{0}" and the argument count: "{1}".</value>
   </data>
+  <data name="MethodGenericArgumentCountException" xml:space="preserve">
+    <value>Could not find a suitable generic method overload for "{0}" with "{1}" type parameters, and the argument count: "{2}".</value>
```

Comment:
I see you follow the file format but I wonder why we put int in quotes.
```suggestion
    <value>Could not find a suitable generic method overload for "{0}" with "{1}" type parameters, and the argument count: {2}.</value>
```

Reply:
I didn't feel like this PR was the place to change that, but I don't disagree 😄

Reply:
Let's keep consistent with the existing error message.